### PR TITLE
Add CSRF prevention

### PR DESCRIPTION
**Pull request by sebastianmarzjan**
This PR adds edittoken handling. Do not merge without accompanying Mercury code ( https://github.com/Wikia/mercury/pull/2113 ) as this will make saving impossible!

### CODE REVIEW

File: extensions/wikia/PortableInfoboxBuilder/PortableInfoboxBuilderHooks.class.php [hunk 1]
```diff
@@ -4,17 +4,6 @@ class PortableInfoboxBuilderHooks {
 	const INFOBOX_BUILDER_SPECIAL_PAGE = 'Special:InfoboxBuilder';
 
 	/**
-	 * Adds infobox builder helper js assets to Template Classification on Edit page
-	 *
-	 * @return true
-	 */
-	public function onTCAfterEditPageAssets() {
```
Comment (rogatty) [inline extensions/wikia/PortableInfoboxBuilder/PortableInfoboxBuilderHooks.class.php:11]: A mistake?

Comment (sebastianmarzjan) [inline extensions/wikia/PortableInfoboxBuilder/PortableInfoboxBuilderHooks.class.php:11]: Yes, we noticed it together with @RafLeszczynski and need to address it

File: extensions/wikia/PortableInfoboxBuilder/PortableInfoboxBuilderHooks.class.php [hunk 2]
```diff
@@ -4,17 +4,6 @@ class PortableInfoboxBuilderHooks {
 	const INFOBOX_BUILDER_SPECIAL_PAGE = 'Special:InfoboxBuilder';
 
 	/**
-	 * Adds infobox builder helper js assets to Template Classification on Edit page
```
Comment (RafLeszczynski) [inline extensions/wikia/PortableInfoboxBuilder/PortableInfoboxBuilderHooks.class.php:7]: this should't be removed

**Comment by RafLeszczynski** [comment]
LGTM except red lines

**Comment by rogatty** [comment]
@SebastianMarzjan is there a reason to not merge it straight to the dev branch?

**Comment by sebastianmarzjan** [comment]
@rogatty No, I just wanted to make it a part of feature branch if OK

**Comment by rogatty** [comment]
Nothing wrong with it but it's an atomic change and could be safely merged separately and from what I heard small PRs are better than big ones ;)

**Pull request merged**